lambda/index.js

```diff
@@ -13,34 +16,39 @@ const MAX_INCIDENT_LENGTH = 250;
 
 export async function handler(event) {  
   const config = JSON.parse(
-    await new AWS.S3().getObject({Bucket: process.env.CONFIG_BUCKET, Key: "config.json"}).promise()
+    await new S3().getObject({Bucket: process.env.CONFIG_BUCKET, Key: "config.json"})
     .then(_=>_.Body.toString("utf-8"))
   );
-  
-  AWS.config.region = config.AWS.region;
 
   const today = new Date();
-  const sts = new AWS.STS();
+  const sts = new STS({
+    region: config.AWS.region
+  });
 
   const data = await sts
     .assumeRole({
       RoleArn: config.AWS.roleToAssume[STAGE],
       RoleSessionName: "lambda-assume-role",
       DurationSeconds: 900
     })
-    .promise()
     .catch(err => {
       console.error("Error assuming cross account role", err);
     });
 
   const stsCredentials = data.Credentials;
-  const assumedCredentials = new AWS.Credentials(
-    stsCredentials.AccessKeyId,
-    stsCredentials.SecretAccessKey,
-    stsCredentials.SessionToken
-  );
-  const dynamo = new AWS.DynamoDB({ credentials: assumedCredentials });
-  const lambda = new AWS.Lambda({ credentials: assumedCredentials });
+  const assumedCredentials = {
+    accessKeyId: stsCredentials.AccessKeyId,
+    secretAccessKey: stsCredentials.SecretAccessKey,
+    sessionToken: stsCredentials.SessionToken
+  };
+  const dynamo = new DynamoDB({
+    credentials: assumedCredentials,
+    region: config.AWS.region
+  });
+  const lambda = new Lambda({
+    credentials: assumedCredentials,
+    region: config.AWS.region
+  });
```

Comment:
Would appreciate any feedback on this change.

I think the result is much the same as using this API described in the docs? Perhaps switching to that would be preferable? https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-credential-providers/#fromtemporarycredentials

Reply:
Why not pass the `stsCredentials` directly?

Reply:
They're different types: the first is of type `Credentials` and the second of type `AwsCredentialIdentity`. They do have different numbers of keys, but the keys also have different names (TitleCase vs camelCase).

I'm also trying to minimise the number of code changes, as ultimately this is a step to getting #50 merged.